cmd/buildah/run.go

```diff
@@ -16,6 +16,11 @@ import (
 var (
 	runFlags = []cli.Flag{
 		cli.StringFlag{
+			Name:  "hostname",
+			Usage: "Set the hostname inside of the container",
+			Value: "localhost",
```

Comment:
A default can be provided in an image, so I prefer to not override it unless we're explicitly asked to.

Reply:
ok